bdb/llmeta.c

```diff
@@ -2682,7 +2677,7 @@ int bdb_add_dummy_llmeta(void)
         logmsg(LOGMSG_ERROR, "%s bdb_tran_commit rc: %d bdberr: %d\n", __func__, rc,
                 bdberr);
         tran = NULL;
-        goto fail;
+        return -1;
```

Comment:
Thank you for catching. We set tran=NULL on failed commit, so goto fail is actually ok.